skills/PRSkill/PullRequestSkill.cs

```diff
@@ -147,9 +149,10 @@ public PullRequestSkill(IKernel kernel)
     {
         var prGenerator = context.Skills.GetFunction(SEMANTIC_FUNCTION_PATH, "PullRequestDescriptionGenerator");
 
-
         var prGeneratorCapture = this._kernel.Skills.GetFunction(SEMANTIC_FUNCTION_PATH, "PullRequestDescriptionGenerator");
-        var prompt = (await prGeneratorCapture.InvokeAsync()).Result;
+        var contextVariablesWithoutInput = context.Variables.Clone();
```

Comment:
Is this still needed or can it be revereted?

Reply:
When the condense skill is used (like it was when I was testing this), it says "match the format of the results as given by the prompt" but the prompt isn't generated with `outputFormatInstructions`, so the final result was not in JSON. This renders the prompt that's sent to CondenseSkill with the outputFormatInstructions but without the git diff input.

Reply:
Ahhh yes, that makes sense! Thanks!